scripts.py

```diff
@@ -553,41 +644,46 @@ def should_process_lane(self, lane):
         # OPDS group feeds are only generated for lanes that have sublanes.
         if not lane.children:
             return False
-        if self.max_depth and lane.depth > self.max_depth:
+        if self.max_depth is not None and lane.depth > self.max_depth:
             return False
         return True
 
-    def do_generate(self, lane):
-        feeds = []
-        annotator = self.app.manager.annotator(lane)
```

Comment:
This is where the problem was -- we were creating an annotator that didn't know which facets were selected.